Doxyfile.in

```diff
@@ -0,0 +1,14 @@
+PROJECT_NAME           = "@CMAKE_PROJECT_NAME@"
+PROJECT_NUMBER         = @VERSION_MAJOR@.@VERSION_MINOR@.@VERSION_PATCH@
+STRIP_FROM_PATH        = @PROJECT_SOURCE_DIR@ \
+                         @PROJECT_BINARY_DIR@
+INPUT                  = @doxy_main_page@ \
+                         @PROJECT_SOURCE_DIR@/src \
+                         @PROJECT_SOURCE_DIR@/include \
+                         @PROJECT_SOURCE_DIR@/doc
+FILE_PATTERNS          = *.hpp \
+                         *.inl \
+                         *.cpp \

```

Comment:
Oh, I haven't seen that the documentation of the `Exception` class was in the `.cpp`, so yeah we need to change that.